### ISSUE TITLE
0.26.13 throws exception "Unable to serialize AsciidocApplicationSettings state"

### ISSUE THREAD

**Issue by twogee** [asciidoctor/asciidoctor-intellij-plugin#266]
```
Unable to serialize AsciidocApplicationSettings state

com.intellij.util.xmlb.XmlSerializationException: Can't serialize instance of class org.asciidoc.intellij.settings.AsciiDocApplicationSettings$State
	at com.intellij.configurationStore.XmlSerializer.serialize(xmlSerializer.kt:63)
	at com.intellij.configurationStore.XmlSerializer.serialize$default(xmlSerializer.kt:47)
	at com.intellij.configurationStore.SaveSessionBaseKt.serializeState(SaveSessionBase.kt:46)
	at com.intellij.configurationStore.SaveSessionBase.setState(SaveSessionBase.kt:20)
	at com.intellij.configurationStore.ComponentStoreImpl.commitComponent(ComponentStoreImpl.kt:283)
	at com.intellij.configurationStore.ComponentStoreImpl.commitComponents$intellij_platform_configurationStore_impl(ComponentStoreImpl.kt:199)
	at com.intellij.configurationStore.ComponentStoreWithExtraComponents.commitComponents$intellij_platform_configurationStore_impl(ComponentStoreWithExtraComponents.kt:89)
	at com.intellij.configurationStore.ComponentStoreImpl.doCreateSaveSessionManagerAndCommitComponents$intellij_platform_configurationStore_impl(ComponentStoreImpl.kt:155)
	at com.intellij.configurationStore.ComponentStoreImpl$createSaveSessionManagerAndSaveComponents$3.invokeSuspend(ComponentStoreImpl.kt:146)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:32)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:236)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:315)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:435)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:419)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:403)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:729)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:678)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:373)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.NullPointerException
	at org.asciidoc.intellij.settings.AsciiDocPreviewSettings.equals(AsciiDocPreviewSettings.java:143)
	at com.intellij.openapi.util.Comparing.equal(Comparing.java:45)
	at com.intellij.util.xmlb.SkipDefaultsSerializationFilter.equal(SkipDefaultsSerializationFilter.java:57)
	at com.intellij.util.xmlb.SkipDefaultsSerializationFilter.equal(SkipDefaultsSerializationFilter.java:25)
	at com.intellij.util.xmlb.BeanBinding.serializePropertyInto(BeanBinding.java:96)
	at com.intellij.util.xmlb.BeanBinding.serializeInto(BeanBinding.java:80)
	at com.intellij.configurationStore.KotlinAwareBeanBinding.serializeInto(xmlSerializer.kt:235)
	at com.intellij.util.xmlb.BeanBinding.serialize(BeanBinding.java:69)
	at com.intellij.configurationStore.XmlSerializer.serialize(xmlSerializer.kt:53)
	... 31 more
```

**Comment by mojavelinux** [comment]
Issue transferred.

**Comment by mojavelinux** [comment]
> since the IntelliJ plugin is using AsciidoctorJ which is using Asciidoctor (Ruby) an issue could potentially track back here.

The plugin is using a mix of AsciidoctorJ and dedicated parser code for IntelliJ, depending on the operation being performed.

**Comment by ahus1** [comment]
@twogee - I'm sorry for this bug. [Version 0.26.14 is available from Github Releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) to fix this issue. Please give it a try. It should be available soon from the JetBrains plugin repository.

**Comment by ahus1** [comment]
... please use 0.26.15 directly, this fixes a bug that was present in 0.26.14.

**Comment by twogee** [comment]
Thanks for quick response.
